### DISCUSSION TITLE
How to get the modified date in doks section?

### DISCUSSION THREAD

**Discussion by cnichte** [gethyas/hyas#454] [Q&A] [answered]
I tried `lastmod` Property  in Frontmatter in combination with `lastMod = true` in `params.toml` but that doesn't have any effect.

**Comment by h-enk** [comment, marked as answer]
You would also need to set `enableGitInfo = true` in `config/_default\hugo.toml`

Undocumented — I will add that. But, should also work without the latter setting I guess — to be fixed.

**Reply by h-enk** [reply]
https://github.com/gethyas/doks-core/issues/89

**Reply by cnichte** [reply]
Thanks that worked perfect :-)
